src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs

```diff
@@ -205,24 +212,27 @@ public static class PermutationFeatureImportanceExtensions
         /// <param name="features">Feature column name.</param>
         /// <param name="useFeatureWeightFilter">Use features weight to pre-filter features.</param>
         /// <param name="topExamples">Limit the number of examples to evaluate on. null means examples (up to ~ 2 bln) from input will be used.</param>
+        /// <param name="permutationCount">The number of permutations to perform.</param>
         /// <returns>Array of per-feature 'contributions' to the score.</returns>
-        public static ImmutableArray<MultiClassClassifierMetrics>
+        public static ImmutableArray<MultiClassClassifierMetricsStatistics>
             PermutationFeatureImportance(
                 this MulticlassClassificationContext ctx,
                 IPredictionTransformer<IPredictor> model,
                 IDataView data,
                 string label = DefaultColumnNames.Label,
                 string features = DefaultColumnNames.Features,
                 bool useFeatureWeightFilter = false,
-                int? topExamples = null)
+                int? topExamples = null,
+                int permutationCount = 1)
         {
-            return PermutationFeatureImportance<MultiClassClassifierMetrics>.GetImportanceMetricsMatrix(
+            return PermutationFeatureImportance<MultiClassClassifierMetrics, MultiClassClassifierMetricsStatistics>.GetImportanceMetricsMatrix(
                             CatalogUtils.GetEnvironment(ctx),
                             model,
                             data,
                             idv => ctx.Evaluate(idv, label),
                             MulticlassClassificationDelta,
                             features,
+                            permutationCount,
                             useFeatureWeightFilter,
                             topExamples);
         }
```

Comment:
(Can't comment on the line six below, so I'll place here)

Missing space:
https://github.com/dotnet/machinelearning/blob/07b8429b5ca4715575c109a3d8313a3d4bd49c42/src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs#L244